components/esp_wifi_remote/scripts/check_headers.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+check_files="
```

Comment:
minor: is this better to define the list of headers for check differently? from separate list file or wifi_remote__build.yml file?

Reply:
Make sense! will add a single source of true

Reply:
I've refactored the script and checker, so It's not needed anymore to keep a list of files to check. I'm doing it in the python script, which only populates the list of files to check when generating them.